### ISSUE TITLE
unable to run EsLint precommit hook

### ISSUE THREAD

**Issue by jonworek** [brigade/overcommit#495]
I have the following setup in my .overcommit.yml file:

```yml
PreCommit:
  EsLint:
    enabled: true
    description: 'Analyze with ESLint'
    required_executable: 'npm'
    flags: ['--format=compact']
    command: ['npm', 'run', 'eslint', '--', '-f', 'compact']
    include: '**/*.js'
```

When I run overcommit on the command line, like so:
`OVERCOMMIT_DEBUG=1 SKIP=HardTabs,TrailingWhitespace,ScssLint overcommit -r`

I get the following output:

```
Running pre-commit hooks
Checking Gemfile dependencies...........................[BundleCheck] FAILED
Gemfile.lock is not up-to-date -- run `bundle check check`
Analyzing with coffeelint................................[CoffeeLint] FAILED
'coffeelint' is not installed (or is not in your PATH)
Install it by running: npm install -g coffeelint
Analyzing with csslint......................................[CssLint] FAILED
'csslint' is not installed (or is not in your PATH)
Install it by running: npm install -g csslint
Skipping HardTabs
Checking for optimizable images..........................[ImageOptim] WARNING
image_optim not installed -- run `gem install image_optim`
Analyzing with JSCS............................................[Jscs] FAILED
'jscs' is not installed (or is not in your PATH)
Install it by running: npm install -g jscs
Analyzing with JSXCS..........................................[Jsxcs] FAILED
'jsxcs' is not installed (or is not in your PATH)
Install it by running: npm install -g jsxcs
Analyzing with JSHint........................................[JsHint] FAILED
'jshint' is not installed (or is not in your PATH)
Install it by running: npm install -g jshint
Validating JSON syntax...................................[JsonSyntax] OK
Analyzing with JSXHint......................................[JsxHint] FAILED
'jsxhint' is not installed (or is not in your PATH)
Install it by running: npm install -g jsxhint
Checking for local paths in Gemfile.............[LocalPathsInGemfile] OK
Checking if database schema is up to date.......[RailsSchemaUpToDate] OK
Analyzing with Rubocop......................................[Rubocop] OK
Skipping ScssLint
Skipping TrailingWhitespace
Checking YAML syntax.....................................[YamlSyntax] OK

✗ One or more pre-commit hooks failed
```

Overcommit is not showing as invoking `eslint`.  Thoughts on what's up?

Thanks!

**Comment by trotzig** [comment]
I can't figure out what's wrong based on what you have provided above. What version of overcommit are you using? When I run with `OVERCOMMIT_DEBUG=1` I get a more verbose output than what you seem to have.

**Comment by jonworek** [comment]
Thanks for the reply, @trotzig.  I *had* 0.22 installed.  Though the strange thing is, I installed the gem fresh from rubygems via my Gemfile on the day that I filed this issue.  So I'm not sure how in the world it would've pulled down such an old version.

But, a `bundle update overcommit` got me the latest when I ran it today.  So now I see eslint running as I expect, and `OVERCOMMIT_DEBUG=1` is also actually having an effect.
